core/tests/protractor/subscriptions.js

```diff
@@ -45,10 +45,10 @@ describe('Subscriptions functionality', function() {
     browser.get(general.USER_PREFERENCES_URL);
     expect(element.all(by.css(
       '.protractor-test-subscription-name')).first().getText()).toMatch(
-      'creator1...');
+      'creator...');
```

Comment:
This is because I have reduced the truncation limit from 11 to 10 so that the usernames don't flow out of the cards. Even though I increased the width of the card, the maximum limit we can achieve is 10. If we increase the width more, the cards come too close to each other.